### PR TITLE
W-16722208: Pom cleanup - remove superfluous configuration

### PR DESCRIPTION
**Pull request by elrodro83**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -259,20 +259,6 @@
...
-                    <executions>
-                        <execution>
-                            <id>analyze</id>
-                            <phase>verify</phase>
-                            <goals>
-                                <goal>analyze</goal>
-                            </goals>
-                        </execution>
-                    </executions>
-                </plugin>
```
Comment (asanguinetti) [inline pom.xml:275]: I don't get what does this have to do with the subject of the PR.

Comment (elrodro83) [reply pom.xml:275]: edited the title

**Review by asanguinetti — APPROVED** [review]

**Comment by elrodro83** [comment]
--validate
